src/discrete-layer/components/job-manager/jobs.dialog.tsx

```diff
@@ -319,12 +319,14 @@ export const JobsDialog: React.FC<JobsDialogProps> = observer((props: JobsDialog
         </DialogTitle>
         <DialogContent className="jobsBody">
           {renderDateTimeRangePicker()}
-          {renderGridList()}
+          {!error && renderGridList()}
           
           {
-            mutationQuery.error !== undefined && (
+            (mutationQuery.error !== undefined || error) && (
```

Comment:
mutationQuery.error  -> shown on the bottom **with** grid
error -> should be shown **instead** of grid